Main/src/Coffee/FindMin.java

```diff
@@ -0,0 +1,30 @@
+package Coffee;
+
+public class FindMin {
+
+
+
+    public static float findMinFrom3Args(float a, float b, float c){
```

Comment:
@GayaneGulanyan jan you can just name `min` , in java we have method overloading and based on args count it understand what to call

Reply:
fixed